test/claim-issuers/claim-to.test.ts

```diff
@@ -0,0 +1,166 @@
+import { expect } from "chai";
+import { ethers } from "hardhat";
+import { BaseContract, ContractFactory } from "ethers";
+import { HardhatEthersSigner } from "@nomicfoundation/hardhat-ethers/signers";
+import { deployIdentityFixture } from "../fixtures";
+import { loadFixture } from "@nomicfoundation/hardhat-network-helpers";
+
+
+interface IIdentity extends BaseContract {
+  getClaim(claimId: string): Promise<{
+    topic: number;
+    scheme: number;
+    issuer: string;
+    signature: string;
+    data: string;
+    uri: string;
+  }>;
+  execute(to: string, value: number, data: string): Promise<any>;
+  addKey(key: string, purpose: number, keyType: number): Promise<any>;
+}
+
+describe("ClaimIssuer", function () {
+  let identity: IIdentity;
+  let owner: HardhatEthersSigner;
+  let other: HardhatEthersSigner;
+
+  beforeEach(async function () {
+    [owner, other] = await ethers.getSigners();
+
+    // Deploy Identity contract
+    const Identity: ContractFactory =
+      await ethers.getContractFactory("Identity");
+    identity = (await Identity.deploy(
+      owner.address,
+      false,
+    )) as unknown as IIdentity;
+    await identity.waitForDeployment();
+
+    const { claimIssuer, aliceWallet } = await loadFixture(
+      deployIdentityFixture,
+    );
+
+    // Add MANAGEMENT key to the identity
+    const claimIssuerKey = ethers.keccak256(
+      ethers.AbiCoder.defaultAbiCoder().encode(
+        ["address"],
+        [await claimIssuer.getAddress()],
+      ),
+    );
+
+    // Add CLAIM key to the identity
+    await identity.addKey(claimIssuerKey, 3, 1); // 3 is CLAIM purpose, 1 is ECDSA type
+    await identity.addKey(claimIssuerKey, 1, 1); // 1 is MANAGEMENT purpose, 1 is ECDSA type
+
+    const identityKey = ethers.keccak256(
+      ethers.AbiCoder.defaultAbiCoder().encode(
+        ["address"],
+        [await identity.getAddress()],
+      ),
+    );
+
+    await claimIssuer.addKey(identityKey, 1, 1); // 1 is MANAGEMENT purpose, 1 is ECDSA type
```

Comment:
why do you add the address of the identity contract as a key 1 and 3 on the claim issuer?